qemu/tests/cfg/chardev_backends.cfg

```diff
@@ -0,0 +1,9 @@
+- chardev_backends: 
+    type = chardev_backends
+    serials += ' vs1 '
+    serial_type_vs1 = virtserialport
+    chardev_backend_vs1 = unix_socket
+    serial_filename_vs1 = /tmp/port0
+    file_transfer_serial_port = vs1
+    guest_script = VirtIoChannel_guest_send_receive.py
+    clean_cmd = 'rm -f'
```

Comment:
This is not a RHEL only case i think @nanliu-r

Reply:
no, we alse test windows